components/accept-financial-contributions/AcceptContributionsMyselfOrOrg.js

```diff
@@ -15,14 +19,46 @@ import StyledInputField from '../StyledInputField';
 import FinancialContributionsFAQ from '../faqs/FinancialContributionsFAQ';
 import MessageBox from '../MessageBox';
 import StripeOrBankAccountPicker from './StripeOrBankAccountPicker';
+import StyledHr from '../StyledHr';
+import CreateCollectiveMiniForm from '../CreateCollectiveMiniForm';
 
 import { withRouter } from 'next/router';
 import { Router } from '../../server/pages';
 import { withUser } from '../UserProvider';
 import { getErrorFromGraphqlException } from '../../lib/errors';
 import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
 
-class AcceptContributionsMyself extends React.Component {
+const CreateNewOrg = styled(Flex)`
+  border: 1px solid lightgray;
+  border-radius: 10px;
+  padding: 20px;
+  cursor: pointer;
+`;
+
+const OrgCard = styled(Flex)`
+  cursor: pointer;
+  border-radius: 10px;
+  &:hover {
+    background: rgba(0, 0, 0, 0.1);
+  }
+`;
+
+const Image = styled.img`
+  @media screen and (min-width: 52em) {
+    height: 256px;
+    width: 256px;
+  }
+  @media screen and (max-width: 40em) {
+    height: 192px;
+    width: 192px;
+  }
+  @media screen and (min-width: 40em) and (max-width: 52em) {
+    height: 208px;
+    width: 208px;
+  }
+`;
```

Comment:
Just mentioning an alternative way to write that with `styled-system`'s [`css` helper ](https://styled-system.com/css) that may prevent typos in the media queries:

```suggestion
const Image = styled.img(
  css({
    width: [192, 208, 256],
    height: [192, 208, 256]
  })
);
```

Reply:
Oh this is much better! Thanks 👍

Reply:
I tried it and it seems the size was not actually applying to the images. I will look further into it in the next PR.